docs/source/api_reference.rst

```diff
@@ -1,8 +1,8 @@
-tensorbaySDK API
-================
+API Reference
```

Comment:
About the commit message, actually this commit changes lots of things like "chapter title", "format", "comment" and so on. I think the word `contents` represents the article body of the docs. Maybe you should find another sentence to describe this commit.